src/ClientGenerator/ClientGenerator.cs

```diff
@@ -102,6 +105,28 @@ private static bool CreateGrainClientAssembly(CodeGenOptions options)
         }
 
         /// <summary>
+        /// Configure logger factory for your client generator
+        /// TODO: fix logging in ClientGenerator, I'm not sure its logging worked properly before, taken that LogManager's state doesn't serialize well across AppDomain
+        /// </summary>
```

Comment:
What does this mean? it is no longer working?

Reply:
I don't think it worked before with LogManager. Since in CreateGrainClientAssembly call it created an app domain and created the GrainClientGenerator in that app domain. I don't think LogManager's state or LogFactory's state will serialize well across that. So I didn't break something, I just mark out that I don't think the logging worked in CientGenerator before